pkg/apis/triggers/v1alpha1/event_listener_validation_test.go

```diff
@@ -153,6 +153,31 @@ func Test_EventListenerValidate(t *testing.T) {
 				),
 				bldr.EventListenerTrigger("tb", "tt", "v1alpha1"))),
 		raiseErr: false,
+	}, {
+		name: "Valid EventListener Multiple Interceptor",
```

Comment:
Would it be valid to have multiple interceptor but of different types e.g. Github + Webhook. Going through the current validation code, it seems like that would result in an error? https://github.com/tektoncd/triggers/blob/master/pkg/apis/triggers/v1alpha1/event_listener_validation.go#L102

It seems like multiple webhooks interceptors should be allowed. Not sure if there is a valid use case for having two Github/Gitlab interceptors though